content/docs/hosted/_index/content.md

```diff
@@ -0,0 +1,18 @@
++++
+fragment = "content"
+weight = 100
+
+title = "Hosted Instance"
+
+[sidebar]
+  enable = true
++++
+
+To easily try out or let us manage the infrastructure, we provide a hosted instance of TXTDirect with global anycasted IPs and a distributed set of servers to redirect your users as fast as possible.
+
+Take a look at our [hosted version](/hosted/) or jump right into the
+[Usage Guide](/docs/hosted/#basics).
+
+- [Basics](#basics)
+- [Subdomains](#subdomains-header)
+- [Subdomains Table](#subdomains)
```

Comment:
I actually tried to write something similar to yc guide but for a self-hosted DNS server. I guess a good thing would be providing separate guides for each DNS provider to make the hosted version as easy as possible to understand and use. It can either be a set of screenshots and descriptions like yc or videos.